src/foolscap/test/test_banana.py

```diff
@@ -1340,12 +1343,12 @@ def testInt(self):
         self.check(-127, bINT(-127))
 
     def testLong(self):
-        self.check(258L, "\x02\x85\x01\x02") # TODO: 0x85 for LONGINT??
-        self.check(-258L, "\x02\x86\x01\x02") # TODO: 0x85 for LONGINT??
-        self.check(0L, "\x85")
-        self.check(0L, "\x00\x85")
-        self.check(0L, "\x86")
-        self.check(0L, "\x00\x86")
+        self.check(long(258), "\x02\x85\x01\x02") # TODO: 0x85 for LONGINT??
+        self.check(long(-258), "\x02\x86\x01\x02") # TODO: 0x85 for LONGINT??
+        self.check(long(0), "\x85")
```

Comment:
Again, not sure if we need long here at all? Why?

Reply:
Here the check was to make sure that both types are handled: if the `isinstance(x, (int,long))` were missing the `long`, then that test would fail, because the object wouldn't be recognized as something serializable.

But in py3, it can go away, because there's only the one integer type. We should be sure to test 258 and -258 and 0, but we don't need to test `int(258)` and `long(258)` separately, so some of those lines can just be deleted. The `L` suffixes can go away too.

Reply:
Done away with the long(...) here and renamed function as testInt2 .